src/executables.js

```diff
@@ -3,6 +3,18 @@
  * @license MPL-2.0
  */
 
+import process from "node:process";
```

Comment:
This should not be imported here, `process` or `process.env` should be provided to `resolveExecutable`. See also #1142.

Reply:
Resolved in ea8d772fbb41b5c585bbcb441fb297830c3466d6